### ISSUE TITLE
 recipe for target 'component-main-build' failed

### ISSUE THREAD

**Issue by V33RU** [hackgnar/ble_ctf#19]
esp/ble_ctf/main/gatts_table_creat_demo.c:25:10: fatal error: esp_gap_ble_api.h: No such file or directory
 #include "esp_gap_ble_api.h"

1. My bluetooth device is enable 
2. and all compiled properly

**Comment by V33RU** [comment]
got the solution its error from the gcc

**Comment by LibertyUnix** [comment]
> got the solution its error from the gcc

What was the fix on your end? I am running into the same issue when trying to build.

**Comment by bitbane** [comment]
I had to make two changes to get this to compile, with a fresh installation of ESP-IDF pulled from github today.

1. Run `make menuconfig` and enable Bluetooth (`Component Config -> Bluetooth -> Bluetooth`).
2. Delete the following two lines from `main/gatts_table_creat_demo.c`
```
static const uint8_t heart_measurement_ccc[2]      = {0x00, 0x00};
static const uint8_t char_value[4]                 = {0x11, 0x22, 0x33, 0x44};
```

These two lines are unused, which throws a warning, and the compiler is running with the option to treat warnings as errors.

With these two fixes I was able to build the CTF.
